master/pkg/tasks/const.go

```diff
@@ -7,6 +7,9 @@ const (
 	taskLoggingSetupScript = "task-logging-setup.sh"
 	taskLoggingSetupMode   = 0744
 
+	taskLoggingTeardownScript = "task-logging-teardown.sh"
+	taskLoggingTeardownMode   = 0755
```

Comment:
Doesn't need to be executable, I believe (and this and the setup script can even be 644 instead of 744).